core/src/main/java/org/infinispan/container/DefaultDataContainer.java

```diff
@@ -350,6 +356,11 @@ public void evict(K key) {
       @Override
       public void onEntryEviction(Map<K, InternalCacheEntry<K, V>> evicted) {
          evictionManager.onEntryEviction(evicted);
+         List<CommandInterceptor> interceptors = interceptorChain.getInterceptorsWhichExtend(CacheMgmtInterceptor.class);
```

Comment:
Also we shouldn't do this call if jmx stats were disabled to begin with - `configuration.jmxStatistics().available()` returns false.